include/libavutil/avutil.pxd

```diff
@@ -13,6 +13,8 @@ cdef extern from "libavutil/avutil.h" nogil:
     cdef char* avutil_configuration()
     cdef char* avutil_license()
 
+    cdef int LIBAVUTIL_VERSION_INT
```

Comment:
Can't we use `avutil_version()`?

Reply:
It seems to be returning this so yes, I guess we can! I'm not familiar with FFmpeg's internals so I missed it. I'll update the PR.

Reply:
Done.